tests.py

```diff
@@ -323,8 +328,32 @@ def test_missing_publickey(self):
         with self.assertRaises(sender.MissingFile):
             ss.send_cpm(0, 0)
 
-    # ...
+  
+class TestDataLog(unittest.TestCase):
+    
+    def setUp(self):
+        print('Checking local data')
+        
+    def test_get_data(self):
+        mgr = Manager(data=True, test=True, interval=10)
```

Comment:
Earlier you had a `test_datalog` file, different from the `global_values` default. That is a good way to do it, so that you can run `tests` without interfering with a real data log file that might already exist.

(Also, if you don't use `mgr.get_interval()` then you don't need to supply an `interval` arg.)